deb/usr/local/bin/ssl_ces.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+source /etc/ces/functions.sh
```

Comment:
There are no functions of the `functions.sh` used here. I think this line should move to the `ssl.sh`.